packages/appstore/src/components/modals/open-positions-svg-modal/index.ts

```diff
@@ -0,0 +1,4 @@
+import OpenPositionsSVGModal from './open-positions-svg-modal';
+import './open-positions-svg-modal.scss';
```

Comment:
a doubt, are we importing styles in index files?
it should be in main component file, right?

Reply:
If you check other Modal files like `account-type-modal` this is the convention we are following. The styles still get applied.